apps/frontend/core-dapp/src/features/portfolio/PortfolioValue.tsx

```diff
@@ -0,0 +1,98 @@
+import { Flex, Icon, spacingIncrement } from 'prepo-ui'
+import styled from 'styled-components'
+import { observer } from 'mobx-react-lite'
+import prepoLogoBgDark from './prepo-logo-bg-dark.svg'
+import prepoLogoBgLight from './prepo-logo-bg-light.svg'
+import { useRootStore } from '../../context/RootStoreProvider'
+import { numberFormatter } from '../../utils/numberFormatter'
+import Skeleton from '../../components/Skeleton'
+
+const { toUsd } = numberFormatter
+
+const Container = styled(Flex)`
+  background: url('${({ theme }): string =>
+      (theme.isDarkMode ? prepoLogoBgDark : prepoLogoBgLight).src}')
+    no-repeat center;
+  background-size: cover;
+  border-radius: ${spacingIncrement(16)};
+  box-shadow: ${({ theme }): string => theme.shadow.prepo};
+  height: fit-content;
+`
+const Label = styled.span`
+  color: ${({ theme }): string => theme.color.neutral3};
+  font-size: ${spacingIncrement(14)};
+  line-height: ${spacingIncrement(14)};
+`
+
+const SmallLabel = styled(Label)`
+  font-size: ${spacingIncrement(12)};
+`
+
+const Value = styled.span`
+  color: ${({ theme }): string => theme.color.secondary};
+  font-size: ${spacingIncrement(16)};
+  font-weight: ${({ theme }): number => theme.fontWeight.semiBold};
+  line-height: ${spacingIncrement(16)};
+`
+
+const BigValue = styled(Value)`
+  font-size: ${spacingIncrement(32)};
+  line-height: ${spacingIncrement(34)};
+`
+
+const ActionLink = styled.a`
+  align-items: center;
+  display: flex;
+  font-size: ${spacingIncrement(12)};
+  font-weight: ${({ theme }): number => theme.fontWeight.semiBold};
+  line-height: ${spacingIncrement(12)};
+`
+
+const PortfolioValue: React.FC = () => {
+  const { portfolioStore, preCTTokenStore } = useRootStore()
+  const { portfolioValue, tradingPositionsValue } = portfolioStore
+  const { tokenBalanceFormat } = preCTTokenStore
+
+  return (
+    <Container p={16} flexDirection="column" alignItems="start" justifyContent="start" gap={24}>
+      <Flex flexDirection="column" alignItems="start" gap={4}>
+        <SmallLabel>Portfolio Value</SmallLabel>
+        {portfolioValue === undefined ? (
+          <Skeleton width={120} height={34} />
+        ) : (
+          <BigValue>{toUsd(portfolioValue)}</BigValue>
+        )}
+      </Flex>
+      <Flex justifyContent="space-between" width="100%">
+        <Flex flexDirection="column" alignItems="start" gap={4}>
+          <Label>Open Positions</Label>
+          {tradingPositionsValue === undefined ? (
+            <Skeleton width={54} height={16} />
+          ) : (
+            <Value>{toUsd(tradingPositionsValue)}</Value>
```

Comment:
ditto can use compactNumber
```suggestion
            <Value>${compactNumber(tradingPositionsValue)}</Value>
```